src/main/java/org/example/springbootintro/controller/BookController.java

```diff
@@ -21,10 +26,16 @@
 @RequiredArgsConstructor
 public class BookController {
     private final BookService bookService;
+    private final BookSearchParametersValidator bookSearchParametersValidator;
 
-    @PostMapping
+    @InitBinder("bookSearchParametersDto")
+    public void initBinder(WebDataBinder binder) {
+        binder.addValidators(bookSearchParametersValidator);
+    }
```

Comment:
Why this is required? Why configuration is performed in controller?

Reply:
I need this to correct working of my own validator: BookSearchParametersValidator